crates/sui-data-ingestion-core/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "sui-data-ingestion-core"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+version = "0.1.0"
+
+[dependencies]
+anyhow.workspace = true
+async-trait.workspace = true
+backoff.workspace = true
+futures.workspace = true
+mysten-metrics.workspace = true
+notify.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+object_store.workspace = true
+prometheus.workspace = true
+telemetry-subscribers.workspace = true
+tokio = { workspace = true, features = ["full"] }
+tracing.workspace = true
+sui-storage.workspace = true
```

Comment:
What do we depend on from sui-storage?

Reply:
we import Blob and BlobEncoding. Maybe we can move them to another crate?

Reply:
no need to do that now, but in the future we may want to explore that in the event that sui-storage depends on rocksdb (which is uneeded for this framework)